modules/social_features/social_activity/src/EmailTokenServices.php

```diff
@@ -247,8 +247,11 @@ public function getUserPreview(User $user) {
     /** @var \Drupal\image\Entity\ImageStyle $image_style */
     $image_style = ImageStyle::load('social_medium');
 
-    /** @var \Drupal\file\FileInterface $image */
-    $image = !$profile->get('field_profile_image')->isEmpty() ? $profile->get('field_profile_image')->entity : '';
+    /** @var \Drupal\file\FileInterface $image|null */
+    $image = NULL;
+    if ($profile->hasField('field_profile_image')) {
+      $image = !$profile->get('field_profile_image')->isEmpty() ? $profile->get('field_profile_image')->entity : '';
```

Comment:
This variable should be FileInterface or Nullable, but if profile image is empty will be a string, change it to Nullable